bibliopixel/animation/animation.py

```diff
@@ -123,12 +118,14 @@ def run_context(self):
         finally:
             self.cleanup()
 
-        if self.__complete_callback:
-            self.__complete_callback(self.complete_reason)
+        if self.on_completion:
```

Comment:
Blame years of C :P It bugs the shit out of me to not have an `if` in there. Makes it harder to read in my opinion.